build.gradle

```
@@ -1,37 +1,74 @@
 plugins {
     id 'com.palantir.docker' version '0.25.0' apply false
+    id 'application'
     id "org.owasp.dependencycheck" version "8.1.2"
 }
 
 subprojects {
     apply plugin: 'java'
     apply plugin: 'com.palantir.docker'
+    apply plugin: 'application'
 
     ext {
         resourceDir = "src/main/resources"
         sharedDir = file("${project.rootDir}/shared")
 
         baseSailfishVersion           = '3.3'
-        sailfishVersion               = "3.3.54"
+        sailfishVersion               = "3.3.106"
```

```suggestion
        sailfishVersion               = "3.3.111"
```
use the last release

---

Done